### ISSUE TITLE
Compare images when testing plot functionality

### ISSUE THREAD

**Issue by f-brinkmann** [pyfar/pyfar#465]
## General

* pyfar version: 0.5.3

## Description

We currently don't compare images when testing plots. This might help:
https://github.com/sunpy/sunpy/blob/a3a076b8e33c445c0d444e33db40ad996f9b242f/sunpy/tests/helpers.py#L86

**Comment by ahms5** [comment]
is this still an issue or is it obsolete?

**Comment by f-brinkmann** [comment]
No, we decided not to compare images automatically and clarified that in the new contributing guidelines
